src/main/java/com/wolox/training/models/Book.java

```diff
@@ -22,10 +25,11 @@
     private String year;
     @Column(nullable = false)
     private int pages;
-    @Column(nullable = false)
+    @Column(nullable = false,unique=true)
     private String isbn;
 
-    public Book(String genre, String author, String image, String title, String subtitle, String publisher, String year, int pages, String isbn) {
+    public Book(String genre, @NotNull String author,@NotNull String image, @NotNull String title,@NotNull String subtitle,
```

Comment:
This validations only work in this constructor, look the validation in this [example](https://www.baeldung.com/javax-validation)